src/lib/server/editors/data/DataSuggestionEditor.ts

```diff
@@ -0,0 +1,22 @@
+import { PromptEditor } from '$lib/server/editors/editor';
+import type { Tables } from '$lib/supabase';
+import { EditorType } from '$lib/types';
+
+export class DataSuggestionEditor extends PromptEditor {
+	constructor(suggestion: string) {
+		super(
+			'DataSuggestion',
+			'Generate Test Cases',
```

Comment:
Imo it would be nice to have the suggestion here as well. I.e. `Generate ${suggestion}`